### PR TITLE
Normalize the namespace of CalcViewModel

### PR DESCRIPTION
**Pull request by tian-lt**
### Description of the changes:
- Root Cause: CalcViewModel is covering itself under namespace CalculuatorApp, which causes a conflicting short name with CalculatorApp.exe that fails the release build every time.
- Changes: Putting all the exported interfaces into namespace CalculatorApp.ViewModel, rename the namespaces according to the file path of each interface.

### Example:

`namespace CalculatorApp::Common::Automation` is changed into
`namespace CalculatorApp::ViewModel::Common::Automation`

### How changes were validated:

- Build passed.
- All the unit test cases are passed locally.
- CI-Pipeline is not applicable at current stage.

### CODE REVIEW

**Review by hanzhang54 — APPROVED** [review]
LGTM

**Comment by tian-lt** [comment]
Hi @wmx16835, feel free to review this PR if you have time.

File: src/CalcViewModel/Common/AppResourceProvider.h [hunk 1]
```diff
@@ -5,16 +5,22 @@
 
 namespace CalculatorApp
 {
-    public ref class AppResourceProvider sealed
+    namespace ViewModel
```
Comment (mingxwa) [inline src/CalcViewModel/Common/AppResourceProvider.h]: Nested namespace could be simplified. For instance, this one could be changed into `namespace ViewModel::Common`.

File: src/CalcViewModel/Common/AppResourceProvider.h [hunk 2]
```diff
@@ -5,16 +5,22 @@
...
-        Windows::ApplicationModel::Resources::ResourceLoader ^ m_stringResLoader;
-        Windows::ApplicationModel::Resources::ResourceLoader ^ m_cEngineStringResLoader;
-    };
+            private:
+                AppResourceProvider();
+                Windows::ApplicationModel::Resources::ResourceLoader ^ m_stringResLoader;
+                Windows::ApplicationModel::Resources::ResourceLoader ^ m_cEngineStringResLoader;
+            };
+        }
+    }
```
Comment (mingxwa) [inline src/CalcViewModel/Common/AppResourceProvider.h]: Add comments indicating the boundary of a namespace declaration, e.g., `// namespace xxx`.

File: src/CalcViewModel/Common/CalculatorButtonPressedEventArgs.cpp [hunk 3]
```diff
@@ -1,11 +1,12 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Copyright (c) Microsoft Corporation. All rights reserved.
```
Comment (mingxwa) [inline src/CalcViewModel/Common/CalculatorButtonPressedEventArgs.cpp]: What's this?

File: src/CalcViewModel/Common/CalculatorDisplay.cpp [hunk 4]
```diff
@@ -8,25 +8,27 @@
...
 
 using namespace CalculatorApp;
+using namespace CalculatorApp::ViewModel;
+using namespace CalculatorApp::ViewModel::Common;
 using namespace CalculationManager;
 using namespace Platform;
 using namespace std;
 
-CalculatorDisplay::CalculatorDisplay()
+CalculatorApp::ViewModel::Common::CalculatorDisplay::CalculatorDisplay()
```
Comment (mingxwa) [inline src/CalcViewModel/Common/CalculatorDisplay.cpp]: Why not writing all the implementations inside the namespace `CalculatorApp::ViewModel::Common`?

File: src/CalcViewModel/Common/CalculatorButtonPressedEventArgs.cpp (hunk 3 above)
Comment (tian-lt) [reply src/CalcViewModel/Common/CalculatorButtonPressedEventArgs.cpp]: UTF-8 BOM caused this diff. reverted back to UTF-8

File: src/CalcViewModel/Common/AppResourceProvider.h (hunk 1 above)
Comment (tian-lt) [reply src/CalcViewModel/Common/AppResourceProvider.h]: Done. Same for all the similar cases.

File: src/CalcViewModel/Common/AppResourceProvider.h (hunk 2 above)
Comment (tian-lt) [reply src/CalcViewModel/Common/AppResourceProvider.h]: Good suggestion. But in the calculator repo, there seems no such practice. Maybe we can create a PR to do such improvements later.

File: src/CalcViewModel/Common/CalculatorDisplay.cpp (hunk 4 above)
Comment (tian-lt) [reply src/CalcViewModel/Common/CalculatorDisplay.cpp]: done

**Review by mingxwa — APPROVED** [review]
LGTM

**Pull request merged**